document.txt

```diff
@@ -0,0 +1,8 @@
+This is an important document 
+-------------
+
+Here's an important list:
+1) Important item

```

Comment:
Testing a reply.

---
In reply to: [54165527](https://github.com/kivancmuslu/CFTest/pull/2#discussion_r54165527) [](ancestors = 54165527)